packages/vscode-extension/src/debug/deleteAadHelper.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { dotenvUtil } from "@microsoft/teamsfx-core/build/component/utils/envUtil";
+import M365TokenInstance from "../commonlib/m365Login";
+import * as globalVariables from "../globalVariables";
+import * as fs from "fs-extra";
+import * as path from "path";
+import { GraphScopes } from "@microsoft/teamsfx-core";
+import axios from "axios";
+import { ConvertTokenToJson } from "../commonlib/codeFlowLogin";
+import VsCodeLogInstance from "../commonlib/log";
+import * as util from "util";
+import { localize } from "../utils/localizeUtils";
+import { ExtTelemetry } from "../telemetry/extTelemetry";
+import { TelemetryEvent } from "../telemetry/extTelemetryEvents";
+import { FxError } from "@microsoft/teamsfx-api";
+
+const defaultNotificationLocalFile = ".notification.localstore.json";
+export async function deleteAad() {
+  try {
+    if (globalVariables.deleteAadInProgress) {
+      return;
+    }
+    globalVariables.setDeleteAadInProgress(true);
+    const projectPath = globalVariables.workspaceUri!.fsPath;
+    const envFile = path.resolve(projectPath, "env", ".env.local");
```

Comment:
The `env` folder name is configurable in teamsapp.yml, please read the folder name from the yaml instead of hard code it.